<?xml version="1.0" encoding="utf-8"?><messages><msg code="0" type="2">Start Time: Tuesday, 15 July 2025 21:09:21</msg><msg code="0" type="0">Starting the generalization process...</msg><msg code="0" type="0">Classifying points...</msg><msg code="0" type="0">Found 424 junction points, 2091 regular points and 26 dead-end points.</msg><msg code="0" type="0">Sorting segments...</msg><msg code="0" type="0">Calling MATLAB function to sort segments...</msg><msg code="0" type="0">MATLAB function call completed.</msg><msg code="0" type="0">checking for self-loops in the sorted segments...</msg><msg code="0" type="0">✅ No self-looping lines found in the initial data. Final sorted data has 804 lines</msg><msg code="0" type="0">Sorting process completed.</msg><msg code="0" type="0">Generalizing sorted segments...</msg><msg code="0" type="0">Dissolved lines created.</msg><msg code="0" type="0">Lines were appended to the generalized lines feature class.</msg><msg code="0" type="0">Midpoints and adj2025 classification were filled in the generalized lines feature class.</msg><msg code="-2147467259" type="100">﻿Traceback (most recent call last):
﻿  File "E:\git\LevellingThesis\Code\GeneralizeSegments.py", line 93, in &lt;module&gt;
﻿    generalize_segments(levelling_segments_input, source_points_input, points_classification_output, sorted_segments_output, generalized_lines_output)
﻿  File "E:\git\LevellingThesis\Code\GeneralizeSegments.py", line 32, in generalize_segments
﻿    generalized_lines = generalize_sorted_segments(sorted_segments, generalized_lines_output, print_messages=True)
﻿                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
﻿  File "E:\git\LevellingThesis\Code\Utils\Generalization.py", line 1090, in generalize_sorted_segments
﻿    EnableEditorTracking(in_dataset=generalized_lines,creator_field="created_user",creation_date_field="created_date",
﻿  File "C:\Program Files\ArcGIS\Pro\Resources\ArcPy\arcpy\management.py", line 9360, in EnableEditorTracking
﻿    raise e
﻿  File "C:\Program Files\ArcGIS\Pro\Resources\ArcPy\arcpy\management.py", line 9343, in EnableEditorTracking
﻿    gp.EnableEditorTracking_management(
﻿  File "C:\Program Files\ArcGIS\Pro\Resources\ArcPy\arcpy\geoprocessing\_base.py", line 533, in &lt;lambda&gt;
﻿    return lambda *args: val(*gp_fixargs(args, True))
﻿                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
﻿arcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.
ERROR 001216: Editor Tracking not supported for dataset GeneralizedLubaRecent
Failed to execute (EnableEditorTracking).

</msg><msg code="0" type="0">Failed script Generalize Segments By Junctions...</msg><msg code="-2147467259" type="100">Failed to execute (GeneralizeSegmentsByJunctions).</msg><msg code="0" type="3">Failed at Tuesday, 15 July 2025 21:14:40 (Elapsed Time: 5 minutes 19 seconds)</msg></messages>